src/debugger/decorations.ts

```diff
@@ -0,0 +1,37 @@
+import * as vscode from 'vscode';
+import * as util from '../utilities';
+
+const instrumentedFunctionDecorationType = vscode.window.createTextEditorDecorationType({
+    borderWidth: '1px',
+    borderStyle: 'solid',
+    overviewRulerColor: 'blue',
+    overviewRulerLane: vscode.OverviewRulerLane.Right,
+    light: {
+        // This color will be used in light color themes
```

Comment:
Minor, minor, but this comment isn't really necessary. =)

Reply:
Carried them over from an example, will remove =)